### PR TITLE
Respect CONVOX_WAIT and --wait on 'convox rack params set'

### PR DESCRIPTION
**Pull request by soulshake**

### CODE REVIEW

**Comment by soulshake** [comment]
[Respect CONVOX_WAIT and --wait on 'convox rack params set'](https://trello.com/c/tVeycToV/2158-respect-convox-wait-and-wait-on-convox-rack-params-set)

File: cmd/convox/rack.go [hunk 1]
```diff
@@ -215,7 +221,19 @@ func cmdRackParamsSet(c *cli.Context) error {
 		return stdcli.Error(err)
 	}
 
-	fmt.Println("OK")
+	if c.Bool("wait") {
+		stdcli.Startf("Waiting for completion")
+
+		// give the rack a few seconds to start updating
+		time.Sleep(5 * time.Second)
```
Comment (MiguelMoll) [inline cmd/convox/rack.go]: nit: Seems like a long wait time. Was two or three seconds tested?

File: cmd/convox/rack.go [hunk 2]
```diff
@@ -215,7 +221,19 @@ func cmdRackParamsSet(c *cli.Context) error {
...
+
+		if err := waitForRackRunning(c); err != nil {
+			return stdcli.Error(err)
+		}
+
+		stdcli.OK()
```
Comment (MiguelMoll) [inline cmd/convox/rack.go]: Think the `fmt.Println("OK")` (or something to that effect) should be incorporated for when wait isn't set.

**Comment by MiguelMoll** [comment]
Rebase would be needed too.

File: cmd/convox/rack.go (hunk 2 above)
Comment (soulshake) [inline cmd/convox/rack.go]: Sorry, I don't see what you mean here. I was trying to use the same style as [here](https://github.com/convox/rack/blob/master/cmd/convox/rack.go#L304-L315), is that not the way to go?

File: cmd/convox/rack.go (hunk 1 above)
Comment (soulshake) [inline cmd/convox/rack.go]: I just used the same sleep time as [here](https://github.com/convox/rack/blob/master/cmd/convox/rack.go#L304-L315). I haven't tested with 2-3 seconds, but can if you think 5 seconds is too long here.

Comment (MiguelMoll) [inline cmd/convox/rack.go]: That's a good example. Thinking something like:

```
stdcli.Startf("Updating parameters... ")
// stuff happens here
stdcli.Wait("OK")
```

Comment (MiguelMoll) [inline cmd/convox/rack.go]: Sounds good to me. 👍

File: cmd/convox/rack.go (hunk 2 above)
Comment (MiguelMoll) [inline cmd/convox/rack.go]: That's a good example. Thinking something like:

```
stdcli.Startf("Updating parameters... ")
// lines 216 -222 go here
stdcli.Wait("OK")
```

Comment (soulshake) [inline cmd/convox/rack.go]: Oh I see. Pushed again, is that better?

**Pull request closed without merging**